core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -172,10 +175,12 @@ class EMConfig {
             val returnType = m.returnType.javaType as Class<*>
 
             if (returnType.isEnum) {
+
                 val elements = returnType.getDeclaredMethod("values")
                         .invoke(null) as Array<*>
 
```

Comment:
this is where you will need reflection to read annotations, and see if any value is marked with `@Experimental`.
likely will need another variable besides `enumValues` to keep track of any found experimental enum